lib/plenario/schemas/virtual_date_field.ex

```diff
@@ -9,6 +9,8 @@ defmodule Plenario.Schemas.VirtualDateField do
 
   alias Plenario.Schemas.VirtualDateField
 
+  @derive {Poison.Encoder, only: [:id, :name, :year_field_id, :month_field_id,
+    :day_field_id, :hour_field_id, :minute_field_id, :second_field_id, :meta_id]}
```

Comment:
I don't think including `meta_id` is very useful -- this is going to be nested under a `Meta` anyway. And similar to the `UniqueConstraint` comment, we could include names here as:

```elixir
field_names = Repo.all(from d in DataSetField, where: d.id in [ ... ], select: {d.id, d.name}) |> Enum.into([])
```

and then access the keyword list by ID to get the names when we encode.